### PR TITLE
Wrap long text without line breaks (#26)

### PR DESCRIPTION
**Pull request by gbanas**

### CODE REVIEW

**Comment by gbanas** [comment]
This commit wraps a text after 35 characters. The screenshot below displays this fix. 


To recreate it locally:
- load 023.0203.json survey in sdx-console,
- enter "http://www.abcdefg.abc/go?id=d0fj10torangy110ShortenURLe1025Redirxb30stringy80100rcG8L0Is.gd0027wrDigBigdrawn%2Bout01drawn%2Bout0EasyURL0c11continueddenlargedoutstretcheda24Fly206enduring41eSnipURLEzURL14NanoRefdistanteh15longishexpandedURL5prolon" value for the 146 key and click the Submit button,
- select Images in sdx-console and load the most recent image.

File: transform/transformers/PDFTransformer.py [hunk 1]
```diff
@@ -99,6 +100,9 @@ def get_table_data(self, question_group):
                 answer = ''
                 if question['question_id'] in self.response['data']:
                     answer = self.response['data'][question['question_id']]
-                table_data.append([Paragraph(question['text'], styleN), Paragraph(answer, styleN)])
+
+                wrapped_answer = "\n".join(wrap(answer, 35))
```
Comment (ajmaddaford) [inline transform/transformers/PDFTransformer.py]: It would be better if this number was defined as a constant e.g. MAX_ANSWER_CHARACTERS_PER_LINE

**Comment by gbanas** [comment]
Updated in https://github.com/ONSdigital/sdx-transform-cs/commit/c9fa2ceb8f569ee76a7180e16d8d25aba95de3b1

**Comment by gbanas** [comment]
Fixed in https://github.com/ONSdigital/sdx-transform-cs/pull/28/commits/83361debdefcfce43afb15b5d95d8d147592eccb

**Pull request merged**